<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Truck</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createTruckPushButoon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create Truck</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="truck_groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>1061</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Truck</string>
    </property>
    <widget class="QLabel" name="chassisWithEngine_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="chassisWithEngine_groupBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>781</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>ChassisWithEngine</string>
     </property>
     <widget class="QLabel" name="chassis_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>251</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="chassis_groupBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>501</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Chassis</string>
      </property>
      <widget class="QLabel" name="frameChassis_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>231</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="tires_label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>231</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="engine_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>251</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="cabin_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>251</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="truck_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>1061</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1362</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
